src/charts/Chart.Radar.js

```diff
@@ -1,15 +1,9 @@
 "use strict";
 
 module.exports = function(Chart) {
-
-	var helpers = Chart.helpers;
-
-	var defaultConfig = {
```

Comment:
yeah, it's local to this function and if it's not being used more than once or severely hurting readability it should be inlined.